lib/utils/comparators.js

```diff
@@ -12,7 +12,8 @@ const operators = {
   empty: (l) => isEmpty(l),
   'not-empty': (l) => !isEmpty(l),
   truthy: (l) => !!l,
-  falsy: (l) => !!!l
+  falsy: (l) => !!!l,
+  'size-of': (l, r) => _.size(l) === r
```

Comment:
Pedantic but I think it should be `size-equals` and maybe even `length-equals`.  I know you're using lodash's size function but i dont think we'd ever plan to use this with an object would we? so its really just checking `(l, r) => l.length === r`

Reply:
I am definitely fine with changing this to not utilize lodash, I used it here to have the additional benefits of automatically handling things which are not 'object-like'.  I think if I change to the above it would also benefit from some light type checking.  Will update the name as well since it is explicitly checking lengths

Reply:
I definitely feel like if we arent type checking the input we should be using lodash here. That being said, do we feel like there is a use case for this to work for strings, etc? If so, using lodash _.size feels like the right call as it handles arrays, objects, and strings.

Reply:
I'm totally cool with lodash i think size is better for type safety., i was more referencing its use discussing `size-equals` vs `length-equals` for the name

Reply:
Sorry misread your comment, sure yeah it makes sense to just alter the name